### DISCUSSION TITLE
How can i get the Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by congthanhngnashtech** [drknzz/GitHub-Achievements#77] [Q&A] [answered]
Does anyone know to get the brain galaxy badge?

**Comment by congthanhng** [comment, marked as answer]
You just answer any Q&A in a discussion topic and the author mark it is the answered. Congrats, you get the badge

**Comment by congthanhngnashtech** [comment]
Thanks @congthanhng. Let's me try

**Comment by congthanhng** [comment]
No problem
